exercise_1/Graph.h

```diff
@@ -23,7 +23,7 @@ class Graph {
     std::vector<Point> _points;
 public:
     Graph(const std::vector<Point> &points);
-    const int calculateVerticals();
+    int calculateVerticals();
```

Comment:
just to be sure:
const int calculateVerticals(); means that you return const value
int calculateVerticals() const; means that the method is const
I asked you to do the second because you do not change the Graph state in this method.

Reply:
The required changes have been implemented